### ISSUE TITLE
OCLint show many 'n errors reported.' messages and generating an empty report.

### ISSUE THREAD

**Issue by LucianoPAlmeida** [oclint/oclint#480]
### Issue Summary
Hi :)
I'm running OCLint on my project and on the sample I created, and it shows a lot of messages of 'n errors reported.' and the report pmd is generated empty.

![screen shot 2017-11-13 at 10 14 41](https://user-images.githubusercontent.com/8292651/32728046-ff1f5f10-c865-11e7-8540-8fc3d96ac6d1.png)

### Environment
- Operation system name and version: macOS Sierra 10.12.6
- OCLint version: 0.12
- How OCLint is installed: local build? prebuilt binary downloaded from github? homebrew install? others? Downloaded file from Github.

### Reproduction Steps
Detailed steps to reproduce the issue.
Run Xcode build:
```sh
xcodebuild -scheme SampleOCLint -workspace SampleOCLint.xcworkspace -configuration Debug | tee xcodebuild.log | xcpretty -r json-compilation-database -o compile_commands.json

```
Run oclint 
```sh
oclint-json-compilation-database -e Pods .
```
My .oclint file:
![screen shot 2017-11-13 at 10 15 18](https://user-images.githubusercontent.com/8292651/32728037-f503b63e-c865-11e7-9fa1-e2efbcf233bf.png)

#### Sample code

Sample project is on github [LucianoPAlmeida/SampleOCLint](https://github.com/LucianoPAlmeida/SampleOCLint). 
Command to run OCLint with the code above.

### Expected Behavior
What do you expect to happen as a result of the reproduction steps?
Generate an oclint.xml PMD report with all the issues.

### Actual Behavior
What currently happens as a result of the reproduction steps?
Is generate an oclint.xml PMD report empty.
```xml
<?xml version="1.0" encoding="UTF-8"?>
<pmd version="oclint-0.12"></pmd>
```
### Even Better
Is your project open sourced? If yes, can you point us to your repository? 
Sample project is on github [LucianoPAlmeida/SampleOCLint](https://github.com/LucianoPAlmeida/SampleOCLint). 

Thank you in advance to look into this :)

**Comment by ryuichis** [comment]
Hi @LucianoPAlmeida, this is a frequently reported issue, and finally, someone is willing to provide a sample project for us to look into. Appreciate that first of all.

However, when I tried your sample, I couldn't even get it to compile without errors. So, if you are open to it, I need your help to set up your sample on Travis CI, and make it fail with `n errors generated`. A good starting point is to derive the `.travis.tml` file from https://github.com/ryuichis/oclint-objc-travis-ci-examples/blob/master/.travis.yml, in fact, if your project is simple enough (which I think so), you should use that file without additional modifications.

If you can report back when you reproduce the same issue on Travis CI, I will definitely take a look. Thanks.

**Comment by LucianoPAlmeida** [comment]
Hey @ryuichis 
Thank you for looking into this, I set up the sample on travis, running on the same conditions that I ran in local machine and in the bitrise CI, you can see this on https://travis-ci.org/LucianoPAlmeida/SampleOCLint/builds/302661356

**Comment by ryuichis** [comment]
Splendid, that is exactly what I am looking for. Thank you first of all. And I have been looking into this, and hopefully will get something back to you later this week. Meanwhile, you can track my progress at my fork: https://github.com/ryuichis/SampleOCLint

**Comment by LucianoPAlmeida** [comment]
@ryuichis Thank you for taking a look into this. Anything you need me to do in the sample or on Travis, just let me know :)

**Comment by TonyYuChufang** [comment]
@ryuichis has any idea to figure out this issue?

**Comment by neufeng** [comment]
add no-error to disable the errors
oclint-json-compilation-database -- -extra-arg=-Wno-error .

**Comment by neufeng** [comment]
update to oclint0.15
copile source 
cd oclint-scripts
python clang prebuilt -release
python build -release
python bundle

**Comment by bettertang** [comment]
/Pods/Headers/Public/AFNetworking/AFURLSessionManager.h | 161:1 | compiler error |   | error | property with 'retain (or strong)' attribute must be of object type
Hi all, has any idea to figure out the issue
